src/main/java/pl/javastart/task/Main.java

```diff
@@ -3,6 +3,9 @@
 public class Main {
 
     public static void main(String[] args) {
-
+        System.out.println(MathUtil.power(3) == 9);
+        System.out.println(MathUtil.circleField(3) == 9*Math.PI);
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj. W instrukcji jest też informacja jak ustawić dla wszystkich nowych projektów. https://bootcamp.javastart.pl/strona/jakosc-kodu